.github/workflows/stage.yaml

```diff
@@ -0,0 +1,85 @@
+name: Stage
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  stage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@master
+
+      - name: build_image
+        id: build_image
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: codebuddies/backend/cb-backend
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          registry: docker.pkg.github.com
+          dockerfile: Dockerfile
+          workdir: cbv3_django_prototype
+          tag_names: true
+
+      - name: start deployment
+        uses: bobheadxi/deployments@master
+        id: deployment
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
+          env: staging
+          transient: true
+          desc: Setting up staging deployment for ${{ steps.build_image.outputs.tag }}
+
+      - name: copy files
+        uses: appleboy/scp-action@master
+        with:
+          host: ${{ secrets.DO_STAGING_HOST }}
+          key: ${{ secrets.DO_SSH_KEY }}
+          passphrase: ${{ secrets.DO_SSH_PASSPHRASE }}
+          port: ${{ secrets.DO_STAGING_SSH_PORT }}
+          username: ${{ secrets.DO_SSH_USER }}
```

Comment:
Are you generally thinking that both `staging` and `production` would be hosted under one DO droplet, but have different DBs?

Reply:
For now, yes. They would run as two containers on the same small droplet. The reason I separated out the secrets is that I believe we may want to run on separate droplets at some point in the future.